sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -147,7 +147,7 @@ const client = new DocumentAnalysisClient("<endpoint>", new DefaultAzureCredenti
 
 - `beginBuildModel` starts an operation to create a new document model from your own training data set. The created model can extract fields according to a custom schema. The training data are expected to be located in an Azure Storage container and organized according to a particular convention. See the [service's documentation on creating a training data set][fr-build-training-set] for a more detailed explanation of applying labels to a training data set.
 - `beginComposeModel` starts an operation to compose multiple models into a single model. When used for custom form recognition, the new composed model will first perform a classification of the input documents to determine which of its submodels is most appropriate.
-- `beginCopyModel` starts an operation to copy a custom model from one Form Recognizer resource to another (or even to the same Form Recognizer resource). It requires a `CopyAuthorization` from the target Form Recognizer resource, which can be generated using the `getCopyAuthorization` method.
+- `beginCopyModelTo` starts an operation to copy a custom model from one Form Recognizer resource to another (or even to the same Form Recognizer resource). It requires a `CopyAuthorization` from the target Form Recognizer resource, which can be generated using the `getCopyAuthorization` method.
 - `getInfo` retrieves information about the resource's limits, such as the number of custom models and the maximum number of models the resource can support.
```

Comment:
Actually one more question, did you have to rename properties on your account properties model?

Also, is the changelog missing?

Reply:
Adding changelog..

> Regarding account props

I was talking to @witemple-msft on this.
The context is that we don't have the same `AccountProperties` type or `getAccountProperties` method in JS. Closest equivalent is this.
```ts
// @public
export interface GetInfoResponse {
    customDocumentModels: CustomDocumentModelsInfo;
}
// @public
export interface CustomDocumentModelsInfo {
    count: number;
    limit: number;
}
```
And the "count" and "limit" are self-explanatory here since  "CustomDocumentModels" is present in the name of the interface and in the name of the parent of these props.

I guess we'll discuss more on this at the naming consistency meeting.